src/main/java/software/amazon/smithy/lsp/SmithyTextDocumentService.java

```diff
@@ -55,9 +55,9 @@
 
 public class SmithyTextDocumentService implements TextDocumentService {
 
-    private List<CompletionItem> baseCompletions = new ArrayList<CompletionItem>();
+    private final List<CompletionItem> baseCompletions = new ArrayList<CompletionItem>();
     private Optional<LanguageClient> client = Optional.empty();
-    private List<Location> noLocations = Arrays.asList();
+    private final List<Location> noLocations = Collections.emptyList();
```

Comment:
Most of the changes in this file can be described by this:
![image](https://user-images.githubusercontent.com/1052965/143074305-b6ea6853-ad93-4c62-a19a-8b71d74863c3.png)

But I won't do it again as it goes againt the internal checkstyle 🙃